src/Components/Confirmation/Confirmation.js

```diff
@@ -530,7 +529,7 @@ const Confirmation = () => {
       return (
         <p key={option} className="bottom-margin">
           {' '}
-          {relatedOptionsList[option]}{' '}
+          {relatedOptionsList[option].message ?? relatedOptionsList[option].props.defaultMessage}{' '}
```

Comment:
I am a little confused how this one works. It looks like you are checking if `relatedOptionsList[option].message` is `undefined` and it is, then you are using `relatedOptionsList[option].props.defaultMessage`. Is the problem that `relatedOptionsList[option]` a `FormattedMessage`? If so, then does this work?
```suggestion
          {relatedOptionsList[option]}{' '}
```